caravel/assets/visualizations/big_number.js

```diff
@@ -6,6 +6,7 @@ require('./big_number.css');
 
 var px = require('../javascripts/modules/caravel.js');
 
+
```

Comment:
weird that that threw errors locally, you could try running `npm install` again in case there are any packages linting depends on that are not installed. or let me know what the errors are.